lib/cpp/src/thrift/endian.h

```diff
@@ -0,0 +1,124 @@
+//
```

Comment:
This header is likely not acceptable for ASF. If you have no objections, please use the standard license header.

Reply:
Since I didn't author the file, but it states that it is in the public domain, or can be licensed under Apache 2.0, should I put the standard Apache license header in addition to the existing text or are you asking me to replace the existing text?

Reply:
Sorry I thought that you were the author. As per [ASF policy](https://www.apache.org/legal/resolved.html#handling-public-domain-licensed-works), let's keep the existing text.